options/locale/locale_en-US.ini

```diff
@@ -3289,6 +3289,18 @@ config.send_test_mail_submit = Send
 config.test_mail_failed = Failed to send a testing email to "%s": %v
 config.test_mail_sent = A testing email has been sent to "%s".
 
+config.incoming_mail_config = Incoming Mail Configuration
+config.incoming_mail_enabled = Enabled
+config.incoming_mail_reply_to_address = Reply To Address
+config.incoming_mail_host = Host
+config.incoming_mail_port = Port
+config.incoming_mail_username = Username
+config.incoming_mail_password = Password
+config.incoming_mail_use_tls = Use TLS
+config.incoming_mail_skip_tls_verify = Skip TLS Verify
+config.incoming_mail_mailbox = Mailbox
+config.incoming_mail_maximum_message_size = Maximum Message Size
```

Comment:
I do not think it is good to translate all these config options.

There are hundreds of them, who would really translate them one by one?